### PR TITLE
feat(mps-sync-plugin): MPS sync plugin inital commit

### PR DESCRIPTION
**Pull request by benedekh**
### TODO

If modelix-core is bumped to 5.0.0 <, then we have to fix https://issues.modelix.org/issue/MODELIX-822.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -48,7 +48,7 @@ subprojects {
     version = rootProject.version
     group = rootProject.group
 
-    val kotlinApiVersion = org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_1_4
+    val kotlinApiVersion = org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_1_6
```
Comment (slisson) [inline build.gradle.kts:51]: This breaks the compatibility with MPS versions before 2022.1. See https://plugins.jetbrains.com/docs/intellij/using-kotlin.html#kotlin-standard-library

Comment (benedekh) [reply build.gradle.kts]: I've reverted it in [bac7b9a](https://github.com/modelix/modelix.mps-plugins/pull/66/commits/bac7b9aa436db8c877f9ba90a11811b857fcd768).

**Review by slisson — APPROVED** [review]
As discussed I didn't review the plugin itself, just checked that it doesn't break existing libraries/plugins.

**Review by slisson — APPROVED** [review]

**Pull request merged**